Cargo.toml

```diff
@@ -18,7 +18,7 @@ members = [
 exclude = [ "example" ]
 
 [dependencies]
-momento-protos = { version = "0.42" }
+momento-protos = { version = "=0.42.4" }
```

Comment:
I don't really think it makes sense to pin to explicit versions like this. Semantically the SDK depends on momento protos 0.42, and the lockfile is the better tool for explicit versions.

I'm not going to block you on this and I will approve once the approvals are green but I don't agree with this style.

Reply:
right now if i don't do this, it literally won't compile because 0.42.5 is not compatible.

Reply:
you could put the version lock in the new cargo.lock file like this

```
[[package]]
name = "momento-protos"
version = "0.42.4"
```

Reply:
One can edit any file one wishes to edit. It's certainly atypical, but it's also quite atypical to publish a library without a cargo.lock file :-(

As I said, I'm not going to block you but I disagree with this plan.